plugins/Marketplace/vue/src/PluginList/HelpLink.vue

```diff
@@ -0,0 +1,31 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <a tabindex="7"
+     class="plugin-details"
+     href="#"
```

Comment:
Should we add a comment that clicking the tile activates the tile link so that we know why it's ok to have `href="#"` here? Same for the MoreDetailsButton perhaps?

Reply:
I'll add a comment that it is only a visual fake link, and the title-link is the actual target.